src/Sql/GdprSqlBase.php

```diff
@@ -83,10 +72,9 @@ public static function getInstance($dbSpec, $options, EventDispatcherInterface $
     }
 
```

Comment:
```suggestion
    // @TODO Combine with above dispatched event to allow changing empty tables?
    if (!empty($config['empty_tables'])) {
      $empty_tables = array_keys(array_filter($config['empty_tables']));
      $options['structure-tables-list'] = $empty_tables;
    }

```

I could be very wrong - as this is totally untested! - but I suspect this might be do the trick to implementing empty tables, as asked in https://github.com/robiningelbrecht/drush-gdpr-dumper/issues/1#issuecomment-453200382 ?

As mentioned in the `@TODO` comment though, it should probably be 'alterable' - maybe the `GdprReplacementsEvent` could expanded to cover more than just replacements, so that both settings could be extended by modules looking to add their own tables?